src/tsolvers/lasolver/LASolver.cc

```diff
@@ -7,12 +7,13 @@
 
 #include "LASolver.h"
 #include "FarkasInterpolator.h"
-#include <simplifiers/LA.h>
 #include "LIAInterpolator.h"
 #include "CutCreator.h"
 
-#include <models/ModelBuilder.h>
+#include <common/numbers/Real.h>
```

Comment:
Isn't this unnecessary?

Reply:
I think not because it is not clear where `Real` comes from. I already experienced a few times that if not included explicitly, and if the order of the other included headers changes e.g. during a refactoring, the compilation may fail and even only on some platforms. Furthermore, it was not trivial to fix such compilation errors.
Hence I consider a better practice to not always rely on indirect inclusions by other headers. Here it would be probably better to move the inclusion into `LASolver.h`, though.